src/en/commands.md

```diff
@@ -464,7 +464,13 @@ Click on the expander to see details for each command.
 
 ^# add-storage
 
-    **Usage:** ` juju add-storage [options] <unit name> <charm storage name> | <charm storage name>=<storage constraints> ... `
+   **Usage:** ` juju add-storage [options] 
+<unit name> <storage directive> ...
```

Comment:
you might want to check this renders okay. This was another bit that script is supposed to rewrite (as above) because it is a) silly and b) breaks the foldout style